gdnative-sys/build.rs

```diff
@@ -65,6 +72,13 @@ mod header_binding {
             }
             _ => {}
         }
+        #[cfg(target_os = "android")]
+        match android_include_path() {
+            Ok(android_include_path) => {
+                builder = builder.clang_arg("-I").clang_arg(android_include_path);
+            }
+            _ => {}
```

Comment:
```suggestion
            _ => panic!("Unable to find android include path")
```

I think there is no point in continuing compilation if the headers are missing, right?